base/client.jl

```diff
@@ -478,6 +478,10 @@ function include(fname::AbstractString)
     Base._include(identity, Main, fname)
 end
 eval(x) = Core.eval(Main, x)
+
+# weakly exposes ans and err and Out variables to Main
+global ans, err, Out
```

Comment:
What's `Out`? Is this also missing docs, like `err`?

Reply:
It is https://docs.julialang.org/en/v1.9-dev/stdlib/REPL/#IPython-mode (but the `Out` variable is not mentioned).